Documentation/Security/GuidelinesAdministrators/FurtherActions.rst.txt

```diff
@@ -27,11 +27,12 @@ Due to the fact that TYPO3 is a PHP application, secure PHP settings
 are also important, of course. PHP settings, such as `open_basedir`,
 `disable_functions` often improve the security of your system and should be considered.
 
-Note that disallowing remote connections (e.g. by deactivating
-`allow_url_fopen` or by blocking outgoing traffic on a firewall in
+In general you are strongly advised to set php.ini settings `allow_url_fopen` and `allow_url_include` to false.
```

Comment:
As I tried to outline before, `allow_url_fopen` and `allow_url_include` are very different things and thus should be treated differently. While it is totally fine and acceptable to rely on `allow_url_fopen` to be set to `true`, it is *not* fine or acceptable to rely on `allow_url_include` to be set to true, nor is it advisable to set this to true in your php.ini

Reply:
In your suggestion, you're putting both in the same bin, which isn't something I'd endorse